crates/egui/src/ui.rs

```diff
@@ -194,6 +202,11 @@ impl Ui {
             ui.set_invisible();
         }
 
+        #[cfg(feature = "accesskit")]
+        ui.ctx().accesskit_node_builder(ui.unique_id, |node| {
```

Comment:
I think it's time we remove the "accesskit" feature, and just always depend on accesskit:

* https://github.com/emilk/egui/issues/3137#issuecomment-1706549867

That is: replace the compile-time flag with a runtime flag. (not in this PR though)